ecommerce-ui/src/PropertyCard/PropertyCard.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import './PropertyCard.css';
+import PropTypes from 'prop-types';
+
+class PropertyCard extends Component {
```

Comment:
It would've made more sense to have PropertyCard component to represent a single property, not a list of properties.